### DISCUSSION TITLE
request to access example

### DISCUSSION THREAD

**Discussion by Dwarakanathan** [DLR-AE/LoadsKernel#4] [Questions & Answers] [answered]
how to access to loads-kernel-examples.git

**Comment by ArneVoss** [comment, marked as answer]
Thank you for your interest in our software! Unfortunately, the examples are take directly from internal DLR projects, so sharing would be very difficult. However, as you are not the only one who asked, an open-source example is on my To Do List. In the meantime, if you need specific help or assistance, we offer commercial support:

- Development of additional, proprietary features
- Consulting & Training courses
- Service & Support

We are interested in partnerships from both industry and academia, so please contact me directly (arne.voss@dlr.de) and we can discuss which simulations you want to do, what kind of models you have, etc.

**Comment by ArneVoss** [comment]
Hey, 
I just wanted to let you know that there are now a number of examples and tutorials available based on the Douglas DC-3 aircraft.
